app/models/spree/order_decorator.rb

```diff
@@ -191,5 +191,46 @@ def flow_payment_method
         'cc' # creait card is default
       end
     end
+
+    def flow_ship_address
+      flow_customer = flow_order&.customer
```

Comment:
To avoid some edge cases, I think we should use flow_order&.destination for flow_ship_address, not customer address.

Reply:
Nice Catch!!! I completely skipped that one!! I will make this change!